process/track/blockProcessor_test.go

```diff
@@ -174,10 +186,15 @@ func TestProcessReceivedHeader_ShouldWorkWhenHeaderIsFromSelfShard(t *testing.T)
 			return nil, nil, nil, nil
 		},
 	}
+	blockProcessorArguments.SelfNotarizer = &mock.BlockNotarizerHandlerMock{
+		GetLastNotarizedHeaderCalled: func(shardID uint32) (data.HeaderHandler, []byte, error) {
+			return &block2.Header{}, nil, nil
```

Comment:
rename package from `block2` to something else

Reply:
done